Firestore/Source/Model/FSTFieldValue.h

```diff
@@ -29,6 +29,25 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+/**
+ * The 'type' of this FSTFieldValue. Used for RTTI (rather than isKindOfClass)
+ * to ease migration to C++.
+ */
+typedef NS_ENUM(NSInteger, FSTFieldValueType) {
```

Comment:
Ideally we wouldn't create this, but instead just use `model::FieldValue::Type` https://github.com/firebase/firebase-ios-sdk/blob/master/Firestore/core/src/firebase/firestore/model/field_value.h#L66

Reply:
Yeah. I was going to do that conversion as part of step two... but it would be pointless. Fixed.